elb_log_ingestor/elb_log_fetcher.py

```diff
@@ -202,7 +207,7 @@ def mark_log_processed(self, logname: str) -> None:
         Move a logfile from the processing to the processed prefix.
         """
         processed_name = self.done_dir / logname.name
-        self.move_object(from_=logname, to=processed_name)
+        #self.move_object(from_=logname, to=processed_name)
```

Comment:
this is an error, but it's the current state - the next line was the one I was supposed to comment 🤦🏻 

This means that we're not currently moving files out of the processing directory, but we _are_ creating empty files in the done directory